### PR TITLE
Change colour on borders

### PR DESCRIPTION
**Pull request by timothyjohn2015**

### CODE REVIEW

File: preset/variables.scss [hunk 1]
```diff
@@ -10,6 +10,7 @@
 $body-font-family: 'proxima-nova', 'helvetica' , sans-serif;
 $heading-font-family: 'freight-sans-pro', 'helvetica', sans-serif;
 $border-radius-root: 5px;
+$btn-letter-spacing: 'unset';
```
Comment (ellerynz) [inline preset/variables.scss:13]: We've removed the letter spacing below: https://github.com/storypark/vue-cli-plugin-vuetify-preset-storypark/blob/master/preset/variables.scss#L133

Comment (timothyjohn2015) [reply preset/variables.scss]: Sweet removed here https://github.com/storypark/vue-cli-plugin-vuetify-preset-storypark/pull/10/commits/6c263fb6b33c5b8f3f67b676188695bebe45f5ed

File: preset/overrides.scss [hunk 2]
```diff
@@ -1,5 +1,12 @@
 @import './variables.scss';
 
+@media (max-width: 588px) {
```
Comment (ellerynz) [inline preset/overrides.scss:3]: 💭 Are there breakpoint variables that we can use?

File: preset/overrides.scss [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
+  .v-expansion-panel-header__icon {
```
Comment (ellerynz) [inline preset/overrides.scss:4]: Is this the type of change we're looking to add to all panel headers now and in the future?

File: preset/overrides.scss (hunk 2 above)
Comment (timothyjohn2015) [reply preset/overrides.scss]: Sweet nice call, I've got this working another way now, including these commits:

https://github.com/storypark/vue-cli-plugin-vuetify-preset-storypark/pull/10/commits/0934e1423a0ba0c87112bd261d7af94005ec33b2

https://github.com/storypark/storyjar/pull/4495/commits/3878845c1573f1cc9ce13e71df42fd4d4e2a4e31

File: preset/overrides.scss (hunk 3 above)
Comment (timothyjohn2015) [reply preset/overrides.scss]: I've changed this now so there's a unique class added on the parent element, which get's the style we need when xsOnly here https://github.com/storypark/vue-cli-plugin-vuetify-preset-storypark/pull/10/commits/0934e1423a0ba0c87112bd261d7af94005ec33b2

**Pull request merged**